src/arith/rewrite_simplify.cc

```diff
@@ -278,10 +278,16 @@ PrimExpr RewriteSimplifier::Impl::VisitExpr_(const AddNode* op) {
     TVM_TRY_REWRITE_IF(floordiv(floormod(x, c2) + c1, c2) + floordiv(x, c2), floordiv(x + c1, c2),
                        c2.Eval()->value > 0);
 
+    TVM_TRY_REWRITE(floormod(x + 1, 2) + floormod(x, 2), OneWithTypeLike(x));
```

Comment:
`TVM_TRY_RECURSIVE_REWRITE_IF(floormod(x + c1, 2), floormod(x, 2) * (-1) + 1, floormod(c1.Eval()->value, 2) == 1);` would lead to `(x + 1) % 2` -> `1 - x % 2` 

Then it seems that rule 281 is already covered, if we rewrite recursively?

Reply:
Good point, and removed.  Testing, it gets the same simplification by first applying the rule on line 289, then 264.